crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -758,6 +761,192 @@ impl RunArgs {
             (None, None) => None,
         }
     }
+
+    pub fn track(&self, telemetry: &CommandEventBuilder) {
+        // track usage
+        if self.cache_dir.is_some() {
+            telemetry.track_arg_usage("cache-dir", true);
+        }
+        if self.continue_execution {
+            telemetry.track_arg_usage("continue", true);
+        }
+        if self.go_fallback {
+            telemetry.track_arg_usage("go-fallback", true);
+        }
+        if self.single_package {
+            telemetry.track_arg_usage("single-package", true);
+        }
+        if self.force.is_some() {
+            telemetry.track_arg_usage("force", true);
+        }
+        // framework_inference defaults to true, so we only track it if it's false
+        if !self.framework_inference {
+            telemetry.track_arg_usage("framework-inference", true);
+        }
+        if self.include_dependencies {
+            telemetry.track_arg_usage("include-dependencies", true);
+        }
+        if self.no_deps {
+            telemetry.track_arg_usage("no-deps", true);
+        }
+        if self.no_cache {
+            telemetry.track_arg_usage("no-cache", true);
+        }
+        if self.daemon {
+            telemetry.track_arg_usage("daemon", true);
+        }
+
+        if self.since.is_some() {
+            telemetry.track_arg_usage("since", true);
+        }
+
+        if self.include_dependencies {
+            telemetry.track_arg_usage("include-dependencies", true);
+        }
+
+        if self.no_deps {
+            telemetry.track_arg_usage("no-deps", true);
+        }
+
+        if self.no_cache {
+            telemetry.track_arg_usage("no-cache", true);
+        }
+
+        if self.daemon {
+            telemetry.track_arg_usage("daemon", true);
+        }
+
+        if self.no_daemon {
+            telemetry.track_arg_usage("no_daemon", true);
```

Comment:
Different casing conventions on this than the other args. Would suggest using a macro to unify how these strings get produced.